src/main/java/domain/LottoRankEvaluator.java

```diff
@@ -13,7 +13,8 @@ public static List<LottoRank> evaluate(Lottos lottos, Lotto winningNumbers, Lott
         }
 
         return lottoList.stream()
-                .map(lotto -> LottoRank.matchRank(lotto.matchNumbers(winningNumbers), lotto.hasBonusNumber(bonusNumber)))
+                .map(lotto -> LottoRank.matchRank(lotto.matchNumbers(winningNumbers),
+                        lotto.hasBonusNumber(bonusNumber)))
```

Comment:
```suggestion
                .map(lotto ->
                        LottoRank.matchRank(lotto.matchNumbers(winningNumbers), lotto.hasBonusNumber(bonusNumber)))
```

요렇게 하는게 좀 더 보기 편하다는 생각이 들었네요
아니면 `.map(lotto -> ...`로 한줄로 만들거나요
딱 `)` 하나만 선이 넘어가는데, 더 좋은 방법이 생각이 안나네요 ㅋㅋ